fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -418,12 +423,26 @@ public PlanFragment visitPhysicalOlapTableSink(PhysicalOlapTableSink<? extends P
             slotDesc.setIsNullable(column.isAllowNull());
             slotDesc.setAutoInc(column.isAutoInc());
         }
-        OlapTableSink sink = new OlapTableSink(
+        checkInnerGroupCommit(context.getScanNodes());
+        OlapTableSink sink;
+        if (context.getConnectContext().isGroupCommitTvf()) {
```

Comment:
should not implement and logic code in translator, translator should only transalte Nereids' plan to legacy one